src/windows/symbol.rs

```diff
@@ -262,25 +266,52 @@ impl RvaSymbols {
         Ok(())
     }
 
+    #[allow(dead_code)]
+    fn filter_public(name: &str) -> bool {
+        if name.starts_with("??_C") {
+            // we've a constant string
```

Comment:
typo: we've got a constant string. You could also replace the comment with a short function with an explicit name like `is_constant_string()`